source/_posts/2019-07-15-complete-flows-partial-models.markdown

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+author: rafa
+title: "(Complete, Flows) -> (Partial) -> (Models)"
```

Comment:
The title seems a little too technical for me. Shouldn't we make it more encouraging? :)